systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/DeploymentUtils.java

```diff
@@ -240,4 +247,20 @@ public static void waitForNoRollingUpdate(String deploymentName, Map<String, Str
 
         return depConfigSnapshot(name);
     }
+
+    public static <T extends Deployment> void logCurrentDeploymentStatus(T deployment) {
```

Comment:
What else here can be here like Deployment?  Please write some documentation explaining what can be T, it will make more sense for the person who will use it after you :).